sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/jms/properties/AzureServiceBusJmsProperties.java

```diff
@@ -69,13 +67,14 @@ public void setTopicClientId(String topicClientId) {
         this.topicClientId = topicClientId;
     }
 
+
     /**
      * Gets the pricing tier.
      *
      * @return the pricing tier
      */
     public String getPricingTier() {
-        return pricingTier;
+        return this.pricingTier;
```

Comment:
Is the idleTimeout used somewhere?

Reply:
yes, it is appended to the amqp remote  uri.